iron-image.html

```diff
@@ -126,13 +141,28 @@
         },
 
         /**
+         * A short text alternative for the image.
+         */
+        alt: {
+          type: String,
+          value: undefined,
+          reflectToAttribute: true
+        },
+
+        _imgDivARIALabel: {
```

Comment:
Yeah, this isn't meant to be set from the outside. It's just part of a bug work-around.